src/otx/core/model/module/detection.py

```diff
@@ -141,4 +141,4 @@ def test_step(self, inputs: DetBatchDataEntity, batch_idx: int) -> None:
     @property
     def lr_scheduler_monitor_key(self) -> str:
         """Metric name that the learning rate scheduler monitor."""
-        return "train/loss"
+        return "val/map_50"
```

Comment:
For the detection recipes, I'll make a separate PR

Reply:
> I think this code change should be included in that PR so it's not confusing. What do you think?
Agree. 

> Is the reason for the separation an experiment?
Since I need to do lots of experiments, I want to take step by step manner.